streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/ProjectService.java

```diff
@@ -56,4 +56,10 @@ public interface ProjectService extends IService<Project> {
 
     boolean checkExists(Project project);
 
+    void updateFailureBuildById(Project project);
+
+    void updateSuccessBuildById(Project project);
+
+    void updateStartBuildById(Project project);
```

Comment:
Three methods can be simplified into one method:

```
void updateBuildById(Long projectId, BuildState state);
```
I recommend to put the updated implementation in `ProjectMapper.xml`